src/django_idom/hooks.py

```diff
@@ -17,7 +17,7 @@
     Callable[..., Callable[..., Awaitable[Any]]],
     _database_sync_to_async,
 )
-WebsocketContext: Context[IdomWebsocket | None] = create_context(None)
```

Comment:
You can use the `|` operator if you add `from __future__ import annotations` to the top of the file.